packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -5,9 +5,6 @@
 import 'dart:async';
 
 import 'package:meta/meta.dart';
-import 'package:test_api/backend.dart'; // ignore: deprecated_member_use
```

Comment:
Do you also need to do this in flutter_platform.dart and tool_coverage.dart ?

Reply:
Done